components/has/overlays/production/kustomization.yaml

```diff
@@ -1,10 +1,20 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - ../base
-  - ../base/external-secrets
+  - ../../base/argocd-roles
+  - ../../base/external-secrets
+  - https://github.com/redhat-appstudio/application-service/config/default?ref=a32c938ee1fcafaed4c64abd3ed4864121fd363c
   - https://github.com/redhat-appstudio/application-service/config/prometheus/?ref=a32c938ee1fcafaed4c64abd3ed4864121fd363c
 
+components:
+  - ../../components/manager-resources
+  - ../../components/enable-profiling
+
+images:
+  - name: quay.io/redhat-appstudio/application-service
+    newName: quay.io/redhat-appstudio/application-service
+    newTag: 05ee33115e4cd7ebfad61d96ffe5ef36c8159dab
```

Comment:
should this tag match line 6 above?

Reply:
yes, similar to what John commented above, I missed this when I had to rebase but now that I have to rebase again, I will make sure I fix this.  Still waiting for https://github.com/redhat-appstudio/infra-deployments/pull/1928 to merge